config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/mechanics/:mechanic_id", to: "mechanics#show"
+  post "/ride_mechanics/:id", to: "ride_mechanics#create"
```

Comment:
If we use the `url` that I mentioned in your `form_with` then your route would be:
```ruby
post "/mechanics/:id/rides", to: "ride_mechanics#create"
```